packages/turf-helpers/index.ts

```diff
@@ -174,7 +174,7 @@ export let areaFactors: any = {
  *
  * //=feature
  */
-export function feature<G = Geometry, P = Properties>(
+export function feature<G extends GeometryObject, P = Properties>(
```

Comment:
This resolves the error `Type 'G' does not satisfy the constraint 'GeometryObject'.ts(2344)`

Turf implementation of Feature didn't specify that the Geometry generic had to have any certain properties (which seems in error), but `geojson` package does, saying that the Geometry generic must be like a `GeometryObject`.  The fix is for downstream generic functions feeding into it to also specify that they extend GeometryObject.

Reply:
actually the latest geojson types do specify a default value, resolvign
```
export interface Feature<G extends Geometry | null = Geometry, P = GeoJsonProperties> extends GeoJsonObject {
```